File: Venmo/src/main/java/com/braintreepayments/api/VenmoLauncherCallback.java
package com.braintreepayments.api;

import androidx.annotation.NonNull;

/**
 * Used to receive notification that the Venmo payment authorization flow completed
 * Once this is invoked, continue the flow by calling
 * {@link VenmoClient#tokenize(VenmoPaymentAuthResultInfo, VenmoTokenizeCallback)}
 */
public interface VenmoLauncherCallback {

    void onVenmoPaymentAuthResult(@NonNull VenmoPaymentAuthResultInfo venmoPaymentAuthResultInfo);
}

File: Venmo/src/main/java/com/braintreepayments/api/VenmoLauncherCallback.kt
package com.braintreepayments.api

/**
 * Used to receive notification that the Venmo payment authorization flow completed
 * Once this is invoked, continue the flow by calling
 * [VenmoClient.tokenize]
 */
fun interface VenmoLauncherCallback {
    fun onVenmoPaymentAuthResult(venmoPaymentAuthResult: VenmoPaymentAuthResult)
}